Mathlib/Data/Complex/Basic.lean

```diff
@@ -814,20 +813,6 @@ noncomputable instance instField : Field ℂ where
 @[simp, norm_cast]
 lemma ofReal_qsmul (q : ℚ) (r : ℝ) : ofReal' (q • r) = q • r := by simp [Rat.smul_def]
 
-section
-set_option linter.deprecated false
-@[simp]
-theorem I_zpow_bit0 (n : ℤ) : I ^ bit0 n = (-1 : ℂ) ^ n := by rw [zpow_bit0', I_mul_I]
```

Comment:
Ref. (brief) discussion in #11414

Reply:
I don't see the connection. I can't replace this lemma by an `ofNat` lemma since the statement depends on the parity.